src/marqo/core/unstructured_vespa_index/unstructured_document.py

```diff
@@ -175,8 +187,18 @@ def to_marqo_document(self, return_highlights: bool = False) -> Dict[str, Any]:
                     json.loads(serialized_multimodal_params)
 
         if return_highlights and self.fields.match_features:
-            if not self.fields.vespa_chunks:
-                raise VespaDocumentParsingError(f"Document {self.fields.marqo__id} does not have any chunks. "
-                                                f"No highlights can be extracted.")
-            marqo_document[index_constants.MARQO_DOC_HIGHLIGHTS] = self.fields.extract_highlights()
+            if self.fields.match_features[f"closest({unstructured_common.VESPA_DOC_EMBEDDINGS})"]["cells"] == {}:
+                # No match feature found, return empty highlights
+                marqo_document[index_constants.MARQO_DOC_HIGHLIGHTS] = []
+            else:
+                if not self.fields.vespa_chunks:
+                    raise VespaDocumentParsingError(f"Document {self.fields.marqo__id} does not have any chunks. "
+                                                    f"No highlights can be extracted.")
+                marqo_document[index_constants.MARQO_DOC_HIGHLIGHTS] = self.fields.extract_highlights()
+
+        # Hybrid search raw scores
+        if self.raw_tensor_score is not None:
+            marqo_document[index_constants.MARQO_DOC_HYBRID_TENSOR_SCORE] = self.raw_tensor_score
+        if self.raw_lexical_score is not None:
+            marqo_document[index_constants.MARQO_DOC_HYBRID_LEXICAL_SCORE] = self.raw_lexical_score
```

Comment:
Seems a bit odd that we have score as a document field. The score coming from Vespa isn't a document field

Reply:
The alternative is to add it as a field in unstructured_schema, but that doesn't seem right either.